examples/smpbe/apbs-smpbe-24dup.in

```diff
@@ -24,7 +24,8 @@ elec name duplexA
    chgm spl2
    srad 1.4
    swin 0.3
-   smpbe vol 10 size 1000
+   # WARN: smpbe is not supported in pbsam-auto
```

Comment:
This wasn't a pbsam-auto calculation, it was an mg-auto calculation.

If we drop smpbe from this example, the example may no longer be useful.

Reply:
Sorry, that should have stated `mg-auto` instead of `pbsam-auto`. The only reference I can find to `smpbe` is in the [Release Notes](https://apbs.readthedocs.io/en/latest/releases.html?highlight=smpbe#id16) from over a decade ago.

Reply:
OK -- I wanted to make sure that we're specifically targeting `mg-*` and `smpbe`.  If we are, then this example can be deleted.

Reply:
There is no reference to smpbe in any of the source code. There is [README](https://github.com/Electrostatics/apbs/blob/master/examples/smpbe/readme.md) in the examples/smpbe directory that states smpbe is currently disabled.  I think it would be safe to remove the examples/smpbe directory. Are you OK with that?

Reply:
I am deleting this example.